controllers/aws_config_loader.go

```diff
@@ -42,15 +41,24 @@ func NewAwsConfigLoaderForEnv(env venv.Env) AwsConfigLoader {
 	}
 }
 
+// CreateNewAWSSessionFromConfig returns an AWS session using AWS Config
+func CreateNewAWSSessionFromConfig(cfg aws.Config) *awssession.Session {
+	sess, _ := awssession.NewSessionWithOptions(
+		awssession.Options{
+			SharedConfigState: awssession.SharedConfigEnable,
```

Comment:
Could you link, in a comment above, to the issue which describes why we need this flag?

Reply:
https://github.com/aws/aws-sdk-go/issues/3101

https://docs.aws.amazon.com/sdk-for-go/api/aws/session/

Reply:
I meant a comment in the code

Reply:
done